sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -697,6 +698,13 @@ public EditEventSteps(
               NAVIGATE_TO_EVENT_DIRECTORY_LIST_GROUP_BUTTON);
         });
 
+    When(
+        "I navigate to EVENT from edit event page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NAVIGATE_TO_EVENT_GROUP_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(NAVIGATE_TO_EVENT_GROUP_BUTTON);
```

Comment:
we can add here another wait for a specific element from form to be displayed, so we won't need to use same wait in other method used after this one